### PR TITLE
chore(protocol): fix issues in test:coverage

### PR DESCRIPTION
**Pull request by xiaodino**

### CODE REVIEW

File: packages/protocol/foundry.toml [hunk 1]
```diff
@@ -67,6 +64,7 @@ test = "test/layer2"
 script = "script/layer2"
 out = "out/layer2"
 evm_version = "shanghai"
+fuzz.runs = 200
```
Comment (dantaik) [inline packages/protocol/foundry.toml:67]: should we also add this config to other prifiles?

File: packages/protocol/test/layer2/LibEIP1559.t.sol [hunk 2]
```diff
@@ -98,23 +98,23 @@ contract TestLibEIP1559 is Layer2Test {
         assertEq(baselineBasefee, basefee);
     }
 
-    /// forge-config: layer2.fuzz.runs = 1000
-    /// forge-config: layer2.fuzz.show-logs = true
+    /// forge-config: layer_2.fuzz.runs = 1000
```
Comment (dantaik) [inline packages/protocol/test/layer2/LibEIP1559.t.sol:101]: why change to "layer_2"?

Comment (xiaodino) [reply packages/protocol/test/layer2/LibEIP1559.t.sol:101]: forge-config will convert layer2 to layer_2 in *.t.sol files when fuzz.runs is used.

Comment (xiaodino) [reply packages/protocol/test/layer2/LibEIP1559.t.sol]: Updated to `forge-config: default.fuzz_runs = 2000`

File: packages/protocol/foundry.toml (hunk 1 above)
Comment (xiaodino) [reply packages/protocol/foundry.toml]: Updated to 
```
[profile.default]
fuzz_runs = 200
```

File: packages/protocol/test/layer2/LibEIP1559.t.sol [hunk 3]
```diff
@@ -98,23 +98,23 @@ contract TestLibEIP1559 is Layer2Test {
         assertEq(baselineBasefee, basefee);
     }
 
-    /// forge-config: layer2.fuzz.runs = 1000
-    /// forge-config: layer2.fuzz.show-logs = true
+    /// forge-config: default.fuzz_runs = 1000
```
Comment (dantaik) [inline packages/protocol/test/layer2/LibEIP1559.t.sol:101]: I think we should also change `layer_2.fuzz_runs` to `default.fuzz_runs`, no?

Comment (xiaodino) [reply packages/protocol/test/layer2/LibEIP1559.t.sol:101]: Good point. Rename both `layer2.fuzz.runs` and `layer2.fuzz.show-logs` to `default.fuzz_runs` and `default.fuzz_runs_show_logs`

**Review by dantaik — APPROVED** [review]

**Review by dantaik — APPROVED** [review]

**Pull request merged**